atoma-state/src/state_manager.rs

```diff
@@ -183,7 +183,6 @@ impl AtomaStateManager {
                                     error = %e,
                                     "Error handling Atoma event"
                                 );
-                                continue;
```

Comment:
Should these be included on this branch?

Reply:
That's totally fine